flow/probes/pcapsocket.go

```diff
@@ -33,17 +33,20 @@ import (
 	"github.com/skydive-project/skydive/config"
 	"github.com/skydive-project/skydive/flow"
 	"github.com/skydive-project/skydive/logging"
+	"github.com/skydive-project/skydive/packetinjector"
 	"github.com/skydive-project/skydive/topology/graph"
 )
 
 // PcapSocketProbe describes a TCP packet listener that inject packets in a flowtable
 type PcapSocketProbe struct {
+	graph     *graph.Graph
 	node      *graph.Node
 	state     int64
 	flowTable *flow.Table
 	listener  *net.TCPListener
 	port      int
 	bpfFilter string
+	injector  *packetinjector.PacketInjector
```

Comment:
is it used ? or is it a remaining thing ?

Reply:
Forgot this one. Removed